enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -325,6 +345,42 @@ The cluster can function without intervention, but to fully restore 4.3, manual
     3. wait for the etcd-member to rejoin
     4. move to the next master.
 
+#### Migrating to IPs
+
+Because reasons.
```

Comment:
Cute, but do better.  Avoiding the need to program DNS, removing a dependency on mDNS, avoiding conflicts between the two, providing resilience during upgrades which disrupt the availability of mDNS to respond to DNS name mappings for rebooting etcd endpoints.